astropy/coordinates/representation/cylindrical.py

```diff
@@ -0,0 +1,164 @@
+##############################################################################
+# IMPORTS
+
+from __future__ import annotations
+
+import operator
+
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates.angles import Angle
+
+from .base import BaseDifferential, BaseRepresentation
+from .base_sphere import _spherical_op_funcs
+from .cartesian import CartesianRepresentation
+
+__all__: list[str] = []
```

Comment:
Why is `__all__` empty? I think it can actually be omitted here, but if included, I'd just do the two classes; one can ensure that the module is not included in the docs (which would make sense).

(Also a bit odd to add typing information here only)

Reply:
I include an empty ``__all__`` to:

1. To show that this module is private. Imports should happen from ``astropy.coordinates`` or ``astropy.coordinates.representation`` (this is included for backwards compatibility).
2. Prevent bad dumps in a namespace via ``from ... import *``.

Python thinks ``__all__`` should be a ``list[str]`` but of course does not enforce it. Empty lists need to be typed to demonstrate compliance with Python expectations.

Reply:
OK on `__all__` - can intent perhaps be even more explicit by adding a comment? Maybe something like
```
# Outside users should import from astropy.coordinates.representations.
```

Reply:
✅